job-descriptions/product-manager.md

```diff
@@ -0,0 +1,74 @@
+
+[Our mission](https://sourcegraph.com/plan) is to dramatically increase the number of people who can understand and write code. By making code more accessible, we will democratize software development and accelerate innovations that bring the future sooner in transportation, health care, energy, AI, communication, space travel, etc.
+
+We value openness and transparency. [Our handbook](https://about.sourcegraph.com/handbook), [our strategy](https://about.sourcegraph.com/company/strategy), and [our code](https://github.com/sourcegraph/sourcegraph) are public. This transparency helps us rapidly gather feedback from our customers so we can iterate, learn, and deliver the best product.
+
+To create a product that serves the needs of all developers, we are building a diverse remote-first team that is distributed across the world. Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
+
+If you are passionate about making the world better through software, come join us!
+
+### About the role
+
+You will help drive the direction of Sourcegraph by collaborating with a small team to solve challenging problems that are fundamental to the growth and success of our business.
+
+Why this role is special:
+
+- Our product has extremely high engagement relative to most products (especially enterprise products). We get enormous amounts of user feedback from our unusually thoughtful user base.
+- Our developers use the product, so they care deeply about product matters. You won't be fighting apathy.
+
+Review our [roadmap](https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit) to understand how our engineering team is organized and our [OKRs](https://about.sourcegraph.com/company/okrs) to understand the goals you will be helping drive.
+
+#### Responsibilities
+
+You will:
+
+- Own an area of the product, and create and drive its product roadmap.
+- Collaborate with engineers and designers to ideate, plan, and build the solutions to customer problems.
+- Use customer feedback to validate priorities, improve documentation, and define product requirements.
+- Be transparent in your internal and external communication. Sourcegraph is open source, and most discussion occurs in public, or in channels where you communicate directly with our customers.
+- Be a passionate advocate for a delightful user experience.
+
+#### Qualifications
+
+- Experience working as a software engineer.
+- Strong understanding of Git and Git workflows.
+- Excellent written and verbal communication skills, technical writing ability, and use of data to back up your arguments.
+- Ability to stay focused on goals and strategically prioritize work.
+- Deep passion for developer tools and productivity enhancements.
+- Demonstrated leadership, the ability to drive product strategy and work well with senior engineers and designers.
+
+#### Nice to haves
```

Comment:
```suggestion
#### Nice-to-haves
```

seems clearer to me

Reply:
I'm fine with this, but for consistency we should update this heading in all the other job postings to have the dashes too. Should I include that in this PR?

Reply:
No, it's not important.